pulsar/win_notify/pillar/beacons.sls

```diff
@@ -0,0 +1,6 @@
+beacons:
+  win_notify:
+    C:/Users:
+
+  interval: 300 # Do not change.  Hard coded into win_notify
```

Comment:
He means his code is hard-coded to only look at events from the last 300 seconds. The interval must still be set for salt, but it must be set to 300 in order to match the hard-coded interval in his code.

We've talked about adding a second interval for the internal setting, since the beacon does not see the interval in its config. (salt strips it out)